src/main/java/duke/entities/Todo.java

```diff
@@ -0,0 +1,41 @@
+package duke.entities;
+
+import java.time.LocalDate;
+
+import duke.enums.TaskType;
+
+/**
+ * Represents the Todo task.
+ */
+public class Todo extends Task {
+
+    /**
+     * Instantiate a todo object.
```

Comment:
small comment, can consider using "Instantiate**s**" instead of "Instantiate" (applies to the serialize() method javadoc as well)